CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Quarkus extension for MicroProfile Language Server Changelog
 
+## [0.19.0](https://github.com/redhat-developer/quarkus-ls/milestone/21?closed=1) (August 14, 2024)
+
+### Enhancements
+
+- Qute: Cannot locate hyphenated template name. See [#975](https://github.com/redhat-developer/quarkus-ls/issues/975).
+- fix: qute(NoMatchingTemplate) if using @CheckedTemplate with basePath. See [#972](https://github.com/redhat-developer/quarkus-ls/pull/972).
+- fix: separate user tag parameters inlay hint. See [#968](https://github.com/redhat-developer/quarkus-ls/pull/968).
+- Qute: add arguments metadata for user-defined tags. See [#928](https://github.com/redhat-developer/quarkus-ls/issues/928).
+- Validate if a BuildItem is `final` or `abstract`. See [#926](https://github.com/redhat-developer/quarkus-ls/issues/926).
+- Support @Route from vertx to display codelens URL. See [#914](https://github.com/redhat-developer/quarkus-ls/issues/914).
+- Support customized template path for @CheckedTemplate. See [#893](https://github.com/redhat-developer/quarkus-ls/issues/893).
+
+### Bug Fixes
+
+- Check if record implement TemplateInstance to provide the support. See [#974](https://github.com/redhat-developer/quarkus-ls/pull/974).
+- Support for Qute arguments. See [#970](https://github.com/redhat-developer/quarkus-ls/issues/970).
+- Qute highlight errors due to formatting . See [#966](https://github.com/redhat-developer/quarkus-ls/issues/966).

```

Comment:
I think this entry could be more specific.

eg. "In qute, allow spaces between method parameters and only validate the return type of the method invocation when iterating over a method invocation"

Reply:
updated